hkmc2/shared/src/test/mlscript/bbml/bbCheck.mls

```diff
@@ -110,7 +110,7 @@ fun baz(z) =
 :fixme // toLoc
 :e
 baz: Int -> (([A] -> A -> A), Int) -> Int
-//│ ╔══[ERROR] General type is not allowed here.
+//│ ╔══[ERROR] Can not instantiate (Int) ->{⊥} (forall α11_2: (α11_2) ->{⊥} α11_2, Int) ->{⊥} Int to mono!.
```

Comment:
It is horribly wrong to try and instantiate a type like the signature here. It's on the right; you can't just remove foralls on the right... Honestly I prefer the previous behavior by default than this unsound nonsense. Again, you didn't try to think hard enough whether what you're doing makes sense, but this time it's worse: it can lead to unsoundness rather than just spurious errors.